plugins/GSdx/GSLocalMemory.cpp

```diff
@@ -41,7 +41,7 @@
 		for(int x = _r.left; x < _r.right; x += w >> 3) \
 		{ \
 			const uint8* src = BlockPtr(_base + off->block.col[x]); \
-			uint8* dst = &_dst[x * bpp]; \
+			dst = &_dst[x * bpp]; \
```

Comment:
We have currently this pattern
```
uint8* _dst = dst;
for () {
  uint8* dst = ...;
  dst += ...;
}
```
For me the affectation inside the loop will only impact the loop variable. The variable before the loop will remain unchanged. I propose to check the behavior with some printf.